src/document.h

```diff
@@ -1065,7 +1065,7 @@ struct Document {
             case A_CASESENSITIVESEARCH: {
                 sys->casesensitivesearch = !(sys->casesensitivesearch);
                 sys->cfg->Write(L"casesensitivesearch", sys->casesensitivesearch);
-                sys->frame->search(sys->searchstring);
+                this->Refresh();

```

Comment:
In case the case sensitivity of the search is toggled, just refresh the document so if one wants to jump to the next matching cells (i.e. the now matched cells) does not have to start from the beginning of the document (F3).